### PR TITLE
Interlineado top ranking

### PR DESCRIPTION
**Pull request by GusBaamonde**
- change of module top ranking

### CODE REVIEW

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js [hunk 1]
```diff
@@ -69,24 +68,26 @@ class ReportsPageRanking extends React.Component {
...
+              <small className="title-ranking">
+                <FormattedMessage id="reports_pageranking.top_pages" />
+              </small>
+              {pages.map((item, index) => (
+                <div key={index} className="page-ranking--item">
+                  <p className="text-ranking"><strong>{index + 1}</strong></p>
+                  <a className="link-ranking" href="" target="_blank">{item.name}</a>
+                  <p className="text-ranking">
+                    <strong>
+                      {item.totalVisits} <FormattedMessage id="reports_pageranking.total_visits" />
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: El texto también va en negrita o solo el número?

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js [hunk 2]
```diff
@@ -69,24 +68,26 @@ class ReportsPageRanking extends React.Component {
...
-          </div>
+        <div className="wrapper-ranking">
+          <div className="reports-box">
+              <small className="title-ranking">
+                <FormattedMessage id="reports_pageranking.top_pages" />
+              </small>
+              {pages.map((item, index) => (
+                <div key={index} className="page-ranking--item">
+                  <p className="text-ranking"><strong>{index + 1}</strong></p>
+                  <a className="link-ranking" href="" target="_blank">{item.name}</a>
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: If the name is the link, it should be in the href.

I have also added a _rel_ property for security reasons.

```suggestion
                  <a className="link-ranking" href={item.name} target="_blank" rel="noopener noreferrer">{item.name}</a>
```

**Comment by andresmoschini** [comment]
❤️

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage } from 'react-intl';
 import Loading from '../../Loading/Loading';
-import './ReportsPageRanking.css';
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: Remember also delete the .css file

Comment (GusBaamonde) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: completely eliminate?

Comment (andresmoschini) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: Of course!

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js (hunk 1 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: just the number

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js (hunk 2 above)
Comment (GusBaamonde) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: Ok

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js [hunk 4]
```diff
@@ -69,23 +68,33 @@ class ReportsPageRanking extends React.Component {
...
+                  <a
+                    className="link-ranking"
+                    href="{item.name}"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {item.name}
+                  </a>
+                  <p className="text-ranking">
+                    <strong>{item.totalVisits}</strong>
```
Comment (andresmoschini) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: It is missed an space, right?
```suggestion
                    <strong>{item.totalVisits}</strong>{' '}
```

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js (hunk 3 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: yes it's only create for the initial styles, no needed

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js [hunk 5]
```diff
@@ -69,23 +67,33 @@ class ReportsPageRanking extends React.Component {
...
+                <FormattedMessage id="reports_pageranking.top_pages" />
+              </small>
+              {pages.map((item, index) => (
+                <div key={index} className="page-ranking--item">
+                  <p className="text-ranking">
+                    <strong>{index + 1}</strong>
+                  </p>
+                  <a
+                    className="link-ranking"
+                    href="{item.name}"
```
Comment (fcoronelmakingsense) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: talk with @andresmoschini it's link to another place?

**Comment by andresmoschini** [comment]
It still looks a little weird:

image

Did you review it with Santiago?

File: src/components/Reports/ReportsPageRanking/ReportsPageRanking.js (hunk 5 above)
Comment (andresmoschini) [reply src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: @fcoronelmakingsense no entendí tu pregunta.

Comment (andresmoschini) [inline src/components/Reports/ReportsPageRanking/ReportsPageRanking.js]: El código correcto es así, como te mostré en [mi comentario anterior](https://github.com/FromDoppler/doppler-webapp/pull/93#discussion_r269709062):

```suggestion
                    href={item.name}
```

En caso contrario lo toma como texto: 

image

**Comment by andresmoschini** [comment]
> Did you review it with Santiago?

@GusBaamonde confirmed to me that Santiago already reviewed it.

**Comment by andresmoschini** [comment]
🚀

**Pull request merged**